src/hooks/useSidePane.ts

```diff
@@ -29,8 +31,9 @@ function useSidePane() {
 
     const [sidePaneNVP] = useOnyx(ONYXKEYS.NVP_SIDE_PANE);
     const [language] = useOnyx(ONYXKEYS.NVP_PREFERRED_LOCALE);
+    const [isAttachmentModalVisible = false] = useOnyx(ONYXKEYS.MODAL, {selector: (modal) => modal?.type === CONST.MODAL.MODAL_TYPE.CENTERED});
```

Comment:
This change is necessary to fix the side pane on large screen:

https://github.com/user-attachments/assets/f85d5b9b-143c-4b63-9e4b-3de7ac78a2b1